Index: Main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import discord\nprint(\"Hello\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Main.py b/Main.py
--- a/Main.py	
+++ b/Main.py	
@@ -1,2 +1,21 @@
 import discord
-print("Hello")
\ No newline at end of file
+import os
+
+client = discord.Client();
+
+
+@client.event
+async def on_ready():
+    print('{0.user} logged in'.format(client))
+
+
+@client.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+
+    if message.content.startswith('$hello'):
+        await message.channel.send('Hello!')
+
+
+client.run(os.getenv('TOKEN'))
